indico/modules/events/contributions/fields.py

```diff
@@ -65,3 +67,19 @@ def pre_validate(self, form):
                 raise ValueError(_("Author data received"))
             if person.author_type == AuthorType.none and not person.is_speaker:
                 raise ValueError(_("{} has no role").format(person.full_name))
+
+
+class SubContributionPersonListField(ContributionPersonListField):
+    """A field to configure a list of subcontribution persons"""
+
+    def _convert_data(self, data):
+        return list({self._get_subcontribution_person(x) for x in data})
+
+    def _get_subcontribution_person(self, data):
+        return SubContributionPersonLink(person=self._get_event_person(data))
+
+    def _serialize_principal(self, principal):
```

Comment:
No need to override this. It should do exactly the same as `ContributionPersonListField._serialize_principal`.